Makefile

```diff
@@ -196,3 +196,11 @@ clean:
 	rm -rf ./tests/e2e/default.proxy
 	rm -rf ./bin/shellcheck*
 	find ./ -name "127.0.0.1:*" -o -name "localhost:*" -o -name "*.log" -o -name "agent-*" -o -name "*.coverprofile" -o -name "testname-proxy-*" -delete
+
+.PHONY: verify-go-versions
+verify-go-versions:
+	./scripts/verify_go_versions.sh
+
+.PHONY: sync-toolchain-directive
+sync-toolchain-directive:
+	./scripts/sync_go_toolchain_directive.sh
```

Comment:
Please also run this script in `./script/fix.sh`, so that contributors don't need to run it separately,.

Reply:
Done

Reply:
Probably I did not comment it clearly, sorry for the confusion. I wasn't saying to run `./scripts/fix.sh` in the target `sync-toolchain-directive`; instead, I was saying to run `./scripts/sync_go_toolchain_directive.sh` inside `./scripts/fix.sh`.

The `./scripts/fix.sh` is the single point of entry to fix everything for etcd.

Reply:
Ah I see, I misunderstood you there, sorry. I've made the change @ahrtr, thank you,